### DISCUSSION TITLE
UI suggestions.

### DISCUSSION THREAD

**Discussion by thezulkifil** [minbrowser/min#1575] [Show and tell]
I have created some new UI designs for Min, I hope to re-design more, such as the website, the logo, but that'll come later! 

![Min UI 2](https://user-images.githubusercontent.com/82689328/115138976-d2396b80-a048-11eb-9b6d-36fa16efa92b.png)
![Min UI 3](https://user-images.githubusercontent.com/82689328/115138978-d82f4c80-a048-11eb-989b-f720f387af58.png)
![Min UI 4](https://user-images.githubusercontent.com/82689328/115138979-d9f91000-a048-11eb-9815-950eb64e9573.png)
![Min UI 1](https://user-images.githubusercontent.com/82689328/115138982-dbc2d380-a048-11eb-8a79-2b5e0a48257e.png)

**Comment by blackgwe** [comment]
👍 for the idea to structure the preferences (`General` / `Privacy` / `View Options` / `key shortcuts` /…); I also like to search within the preferences view.
* The url within the tab view I dislike. (Always press `cmd` + `l` to enter edit mode and view / change url)
* I don't need the `reload icon` as shown in 4th image, it's waste of space: Maybe it can be shown when hovering the tab?
* 2nd image: I do not understand what you mean. Please explain it - Thanks!

Another nice feature could be to "pin tabs" = show favicon instead of the title and optionally access them e.g. with `cmd` + `1` /  `cmd` + `2` and so on… (as a contribution for a cleaner minimal user interface)

**Reply by thezulkifil** [reply]
Hi, This was meant to be an animation so the features are explained a bit better but due to some issues I was unable. The link and reload icon will only appear once you double click the tab so after that you can the URL, search, etc
The 2nd image is suppose to be a homepage of sorts, It will show your tasks so they are more accessible.

**Comment by PalmerAL** [comment]
Sorry for the delay, I just had time to look at this. Initial thoughts:

- I like the sidebar for the preferences screen.
- I don't like they gray color scheme; the text seems harder to read than the current version, and I don't think it looks as nice.
- I don't think showing the search bar all the time is a good idea. What could work, however, is opening it when the tab is clicked, but not having it expand to the full width (so that other tabs are still visible). Would probably need to build a prototype to see if this is actually a good idea of not.
- On Windows, it's not possible to have a region be both clickable and move the window when dragged, so you need to leave a blank space somewhere for dragging the window (right now, this is a blank strip on top of the tabs).

But before we get any further with this, I think it would be a good idea to figure out what the overall goal is; I don't understand the reasoning for these changes, which makes it hard for me to assess them. What issues do you see with the current UI that you're trying to address? Once we've identified those, it should be easier to figure out what the right solutions are.

**Reply by thezulkifil** [reply]
It's all good mate!

1. Well yes, It was just there so everything is readable and matches the website/the original UI.
2. I agree, It'll be hidden unless someone double clicks on the tab and than the URL appears. 

The space isn't used correctly the icons are a bit too big and the UI feels a bit clustered, I'd like it to be simple, sleek and minimal, and a better dark mode, something dark grayish, And I'll try and get the animation ready and this isn't close to the final version, call it a early concept!
